Makefile

```diff
@@ -24,7 +24,7 @@ mypy:  ## Run mypy
 
 .PHONY: unittest
 unittest: ## Run unittest
-	python -m pytest test/unit/
+	python -m pytest ${ARGS} test/unit/
```

Comment:
`$ARGS` or `@{ARGS}` ?

Reply:
at least it can be `$ARGS` (not sure `@{ARGS}`), but will use `$(ARGS)` as well as others in this script.